src/Objects/Schema/Generation/SchemaGenerator.php

```diff
@@ -53,78 +59,26 @@ private function generateFromClass(ReflectionClass $class, Type $type): Schema
         return $schema;
     }
 
-    private function schemaFromTypes(Type ...$types): Schema
+    /**
+     * @param array<Type> $types
+     */
+    private function schemaFromTypes(array $types): Schema
```

Comment:
Could you explain to me why you changed the signature to a simple array over using the variadic type argument?

```php
private function schemaFromTypes(Type ...$types): Schema
```

Reply:
Simply to avoid unpacking the arrays when passing arguments to this function. But thinking better about this, we loose a more strict type checking by doing so. Let me revert this change.